infra/cifuzz/cifuzz_external_entrypoint.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 Google LLC
```

Comment:
nit: Can you rename this file `cifuzz_combined_entrypoint`. I'm not sure it's going to be the preferred method for users that can't build two docker images.

Reply:
done